Source/FlowEditor/Private/Asset/FlowAssetEditor.cpp

```diff
@@ -358,6 +358,10 @@ void FFlowAssetEditor::BindToolbarCommands()
 								FExecuteAction::CreateSP(this, &FFlowAssetEditor::ValidateAsset_Internal),
 								FCanExecuteAction());
 
+	ToolkitCommands->MapAction(ToolbarCommands.EditAssetDefaults,
```

Comment:
Adding the Edit Asset Defaults button

Reply:
How we could live without it? xD

Reply:
Update: moved the Asset Defaults button after Diff and Search buttons, this is more consistent with a blueprint toolbar :)